SI/Sem.02/Pract.02/task_7_p02.cpp

```diff
@@ -0,0 +1,30 @@
+
+
+#include <iostream>
+
+int main()
+{
+    char symbol;
+    std::cout << "Enter a letter: ";
+    std::cin >> symbol;
+    char LowerToUpper = symbol - 32;
+    char UpperToLower = symbol + 32;
```

Comment:
You should not do these operations in advance when you will need only one of them
You should do these operations when you enter the if for LowerToUpper
Also, use camelCase for naming you variables